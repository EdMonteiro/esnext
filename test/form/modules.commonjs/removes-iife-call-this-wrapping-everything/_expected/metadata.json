{
  "modules.commonjs": {
    "imports": [],
    "exports": [],
    "directives": [],
    "unwrapped": {
      "type": "FunctionExpression",
      "id": null,
      "params": [],
      "body": {
        "type": "BlockStatement",
        "directives": [],
        "body": [
          {
            "type": "IfStatement",
            "test": {
              "type": "Identifier",
              "name": "a",
              "typeAnnotation": null
            },
            "consequent": {
              "type": "BlockStatement",
              "directives": [],
              "body": [
                {
                  "type": "ExpressionStatement",
                  "expression": {
                    "type": "CallExpression",
                    "callee": {
                      "type": "Identifier",
                      "name": "b",
                      "typeAnnotation": null
                    },
                    "arguments": []
                  }
                }
              ]
            },
            "alternate": null
          }
        ]
      },
      "generator": false,
      "async": false,
      "returnType": null,
      "typeParameters": null
    }
  }
}