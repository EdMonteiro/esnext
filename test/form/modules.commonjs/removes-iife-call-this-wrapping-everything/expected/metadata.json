{
  "modules.commonjs": {
    "imports": [],
    "exports": [],
    "directives": [],
    "unwrapped": {
      "type": "FunctionExpression",
      "id": null,
      "params": [],
      "body": {
        "type": "BlockStatement",
        "body": [
          {
            "type": "IfStatement",
            "test": {
              "type": "Identifier",
              "name": "a"
            },
            "consequent": {
              "type": "BlockStatement",
              "body": [
                {
                  "type": "ExpressionStatement",
                  "expression": {
                    "type": "CallExpression",
                    "callee": {
                      "type": "Identifier",
                      "name": "b"
                    },
                    "arguments": []
                  }
                }
              ]
            },
            "alternate": null
          }
        ]
      },
      "generator": false,
      "expression": false
    }
  }
}